### ISSUE TITLE
Terminal accessibility buffer does not read output upon enter

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#172007]
Terminal live output is not announced.
As far as I tested on Windows with JAWS and NVDA, terminal aria-live event is not auto-announced. I typed "echo hello" in terminal, but there was no read-aloud feedback.

https://github.com/microsoft/vscode/issues/171914#issue-1551817935

**Comment by meganrogge** [comment]
@jooyoungseo, since the buffer will now be readonly, I think this can be closed. Voice over is announcing as I type in the normal terminal (not accessibility buffer)

**Comment by jooyoungseo** [comment]
@meganrogge  Aw, I think there was a miscommunication between us. The issue that I highlight here is not the input announcement. I am talking about the auto output announcement. For example, if you open an external terminal on Mac or cmd.exe or powershell.exe on Windows outside VSCode, executing any command, such as `echo hello` will make your screen reader to read aloud the output as soon as you hit return key. The VSCode terminal was behaving like this before the a11y temrinal buffer. What you have seen is that screen-reader users want this real-time output announcement per command execution just like before. This no longer works as far as I tested with NVDA and JAWS on Windows.

**Comment by meganrogge** [comment]
Thanks for clarifying. I was confused by:

> I typed "echo hello" in terminal, but there was no read-aloud feedback.

Now I understand

**Comment by meganrogge** [comment]
I can repro this not working in the latest stable VS Code as well. I type `echo hello` and it does not read `hello`

**Comment by jooyoungseo** [comment]
@meganrogge  -- Aw, that's my bad! Sorry for confusing you. I assume that `aria-live = "assertive"` attribute got broken somewhere for the terminal a11y buffer.

**Comment by meganrogge** [comment]
@jooyoungseo I have just spent the last 2 hours looking into the output not being read issue and am stumped 🤔 . since I can repro in Stable, I don't think this is a regression. We're doing the right thing here and debugging showed the element's text content is updated to be the output on enter, but the browser /screen reader is not reading it

https://github.com/meganrogge/xterm.js/blob/01b0eb615c0491135c350a28553ff947a266f0fa/src/browser/AccessibilityManager.ts#L83-L86

**Comment by jooyoungseo** [comment]
@meganrogge -- I would try the following:

1. See if aria-live works when terminal buffer container is marked with `div` instead of `textarea` (also need to remove role = "textbox" for this testing).

2. If 1. works, just add textbox role and see if it works.

**Comment by meganrogge** [comment]
@jooyoungseo thanks for the suggestion. still does not work with that. I'm going to try to find the last commit where this was working - which will be at least a month ago

**Comment by meganrogge** [comment]
in version 1.72 of VS Code stable, it similarly does not work for short output, but does read the message "too much output to announce for the command `ls -la` for example.

**Comment by meganrogge** [comment]
also just tried reverting all of my changes (adding the accessibility buffer) and that does not fix it

**Comment by meganrogge** [comment]
navigation mode also works in 1.72

**Comment by meganrogge** [comment]
the `.xterm-accessibility` element doesn't exist in insider's with screen reader mode on while it does exist in version 1.72

<img width="935" alt="Screenshot 2023-01-23 at 2 53 35 PM" src="https://user-images.githubusercontent.com/29464607/214147688-5dd538c3-4369-4f11-9d77-564c1a160e3b.png">

<img width="1247" alt="Screenshot 2023-01-23 at 2 53 15 PM" src="https://user-images.githubusercontent.com/29464607/214147817-1013c51c-1843-4225-a507-222f0fdb08c6.png">
